API/Tensorflow/native_libs/src/c_wrappers/tensors.cpp

```diff
@@ -52,6 +53,91 @@ TFL_API int64_t get_tensor_flatlist_length(Tensor* tensor, const char **outError
     };
 }
 
+static inline void write_int64(std::ostream& binary_stream, int64_t x) {
```

Comment:
It would be more consistent with the rest of our code to put these functions in an anonymous namespace.